src/main/resources/logback.xml

```diff
@@ -7,9 +7,9 @@
             </pattern>
         </encoder>
     </appender>
-    <appender name="appender" class="me.karakelley.http.Appender"/>
+    <appender name="appender" class="me.karakelley.http.utility.Appender"/>
```

Comment:
Does this have to be declared in the logback.xml? Is this specific appender implementation specific to tests?